Source/WebCore/css/calc/CSSCalcTree+Serialization.cpp

```diff
@@ -241,7 +242,7 @@ static Vector<ChildRepresentation, 16> generateSortedChildrenMap(const Children&
     for (size_t i = 0; i < children.size(); ++i)
         sortedChildrenMap.append(ChildRepresentation { .index = i, .sortPriority = sortPriority(children[i]) });
 
-    std::stable_sort(sortedChildrenMap.begin(), sortedChildrenMap.end(), [](const auto& first, const auto& second) -> bool {
+    std::ranges::stable_sort(sortedChildrenMap, [](const auto& first, const auto& second) -> bool {
```

Comment:
```suggestion
    std::ranges::stable_sort(sortedChildrenMap, [](auto& first, auto& second) {
```

Reply:
I haven't really used projections much at all, but I wonder if something like this would be more clear written as:

```c++
std::ranges::stable_sort(sortedChildrenMap, {}, &ChildRepresentation::sortPriority);
```

Reply:
I was planning to look into potentially using projections in a follow-up since this is more involved and a bit riskier. I figured it was best to keep the non-trivial stuff out of the large refactoring PRs.